diff --git a/node_modules/oc-azure-storage-adapter/lib/index.js b/node_modules/oc-azure-storage-adapter/lib/index.js
index b69e967..0f03f79 100644
--- a/node_modules/oc-azure-storage-adapter/lib/index.js
+++ b/node_modules/oc-azure-storage-adapter/lib/index.js
@@ -40,7 +40,8 @@ function azureAdapter(conf) {
     });
     const getClient = () => {
         const sharedKeyCredential = new storage_blob_1.StorageSharedKeyCredential(conf.accountName, conf.accountKey);
-        const blobServiceClient = new storage_blob_1.BlobServiceClient(`https://${conf.accountName}.blob.core.windows.net`, sharedKeyCredential);
+        const blobUrl = conf.blobUrl || `https://${conf.accountName}.blob.core.windows.net`;
+        const blobServiceClient = new storage_blob_1.BlobServiceClient(blobUrl, sharedKeyCredential);
         return blobServiceClient;
     };
     const getFile = async (filePath, force = false) => {
@@ -54,6 +55,7 @@ function azureAdapter(conf) {
                 return fileContent;
             }
             catch (err) {
+                console.error(err);
                 if (err.statusCode === 404) {
                     throw {
                         code: oc_storage_adapters_utils_1.strings.errors.STORAGE.FILE_NOT_FOUND_CODE,
